optaplanner-examples/src/test/java/org/optaplanner/examples/vehiclerouting/optional/score/VehicleRoutingConstraintProviderTest.java

```diff
@@ -47,7 +47,7 @@ public void vehicleCapacityUnpenalized() {
 
         constraintVerifier.verifyThat(VehicleRoutingConstraintProvider::vehicleCapacity)
                 .given(vehicleA, customer1)
-                .hasNoImpact();
+                .penalizesLong(0);
```

Comment:
This is the main reason why penalizesBy() can't use overloading for penalizesBy(long).